golden/clr-angular.d.ts

```diff
@@ -471,10 +473,7 @@ export declare enum ClrDatagridSortOrder {
 }
 
 export interface ClrDatagridStateInterface<T = any> {
-    filters?: ({
```

Comment:
As discussed, this is a bummer but I'm not sure we can do better right now. 😞

Reply:
😢